src/Appwrite/Event/Event.php

```diff
@@ -24,11 +24,22 @@ class Event
     public const FUNCTIONS_QUEUE_NAME = 'v1-functions';
     public const FUNCTIONS_CLASS_NAME = 'FunctionsV1';
 
+    /** remove */
     public const USAGE_QUEUE_NAME = 'v1-usage';
     public const USAGE_CLASS_NAME = 'UsageV1';
 
     public const USAGE_DUMP_QUEUE_NAME = 'v1-usage-dump';
     public const USAGE_DUMP_CLASS_NAME = 'UsageDumpV1';
+    /** /remove */
+
+    public const STATS_RESOURCES_QUEUE_NAME = 'v1-stats-resources';
+    public const STATS_RESOURCES_CLASS_NAME = 'StatsResources';
```

Comment:
The others have V1 in name

Reply:
I think this is long overdue removal, this class name is not seem to be used but I might be wrong.